controllers/countriesControllers.js

```diff
@@ -0,0 +1,77 @@
+const countriesRepositories = require('../repositories/countriesRepositories');
+const uuid = require('uuid'); //uuid needs to be installed
+const countriesUtilities = require('../utilities/countriesUtilities');
+
+
+//get all countries
+exports.getAllCountries = (req, res) => {
+    res.success(countriesRepositories.getAllCountries());
+}
+
+//get a single country
+exports.getCountry = (req, res) => {
+    const availableCountries = countriesRepositories.getCountry(req.params.name);
+    if (availableCountries.length !== 0) {
+        res.success(availableCountries);
+    }
+    else {
+        const countryName = countriesUtilities.capitalizeFirstLetter(req.params.name);
+        return res.notFound();
+    }           
+};
+
+// post/add a country
+exports.postCountry = (req, res) => {
+
+            const newCountry = {
+                id: id = uuid.v4(), 
```

Comment:
nu cred ca este nevoie de `id =`, sau daca neaparat vrei sa salvezi acel uuid intr-o alta variabile si nu doar ca si un property la object-ul de newCountry atunci ii preferabil sa fie inainte de object declaration si nu in el

Reply:
intradevar, nu era nevoie de un id aici. L-am folosit doar ca sa simulez o baza de date reala, in care s-ar genera un id cand ar fi postata o tara noua. Multumesc pentru feedback! Am facut modificarile, arata mai clean codul cu destructuring assignment